client/src/main/java/io/pravega/schemaregistry/client/SchemaRegistryClientImpl.java

```diff
@@ -486,4 +507,40 @@ public void close() throws Exception {
             client.close();
         }
     }
+
+    private SSLContext getSSLContext(SchemaRegistryClientConfig config) {
+        try {
+            KeyStore trustStore;
+            if (config.isSystemPropTls()) {
```

Comment:
This property should be removed. Absence of an explicitly specified truststore should automatically lead to fallback to the system's truststore.